translations/ui-data-import/en.json

```diff
@@ -141,6 +141,8 @@
   "fileDefinitionDeleteError": "Internal error. Press {button} button to try again",
   "noUploadedFiles": "No files to show",
   "jobProfileName": "Job profile",
+  "jobProfileNameLink": "Job profile: {jobProfileLink}",
+  "jobHrId": "Job {hrId} • ",
```

Comment:
Let's use the middot symbol `&middot` as it's done in `MatchColumn` listTemplate